subprojects/publish/src/test/groovy/org/gradle/api/publish/internal/metadata/GradleModuleMetadataWriterTest.groovy

```diff
@@ -1160,6 +1160,90 @@ class GradleModuleMetadataWriterTest extends Specification {
 """
     }
 
+    // fixing a problem reported in https://github.com/gradle/gradle/issues/19769
```

Comment:
Use `@Issue` for this, there is an example in the same file line 878

Reply:
Done